docs/source/qvm.rst

```diff
@@ -36,9 +36,9 @@ The QVM is available on your local machine. You can initialize a localQVM instan
 
 One executes quantum programs on the QVM using two paradigms: the ``.run(...)`` method, and
 the ``.wavefunction(...)`` method. The former closely mirrors how one will execute programs on a
```

Comment:
Understandable if this isn't in scope for this PR, but the QVM does not have a `.wavefunction()` method